### DISCUSSION TITLE
Chat Buttons to kick off a prompt or another flow

### DISCUSSION THREAD

**Discussion by mdwoicke** [FlowiseAI/Flowise#584] [Ideas]
You guys rock! Love the new feature for showing source documents for a QA search! I was wondering if it could be possible to have this button functionality to create preset options like some chatbots have to either answer a question with a preset value to to kick off a different flow altogether.

**Comment by HenryHengZJ** [comment]
for now, you can't do that yet, but this is one of the features we have in the roadmap!
